cvat/apps/organizations/serializers.py

```diff
@@ -126,12 +125,7 @@ def create(self, validated_data):
             user_email = membership_data["user"]["email"]
             user = User.objects.create_user(username=user_email, email=user_email)
             user.set_unusable_password()
-            # User.objects.create_user(...) normalizes passed email and user.email can be different from original user_email
-            email = EmailAddress.objects.create(
-                user=user, email=user.email, primary=True, verified=False
-            )
```

Comment:
So if I'm understanding this change correctly, previously a dummy user would have an associated `EmailAddress` object with `verified=False`, while now a dummy user will have no associated `EmailAddress` at all.

That seems reasonable to me (and the implementation seems much simpler, which is nice). However, what happens with users for whom an `EmailAddress` has already been created but who have not yet accepted the invitation? Will they not be able to accept anymore? Can we fix this with a migration?

Reply:
However, I agree that we need to add a migration to fix possible mismatches between `EmailAddress.email` and `User.email`.

Reply:
OK, but let's suppose a person (who is not yet a user) is invited into an organization. A dummy user with an attached `EmailAddress` gets created for them. Then CVAT is upgraded to the new version. Then the person tries to register. IIUC, `get_dummy_user` will no longer return the previously created dummy user, since it now looks for users with no attached `EmailAddress`. So `RegisterSerializerEx` will create a new user, which is incorrect. Am I wrong?